user-org-actor/src/main/java/org/sunbird/user/service/AddressServiceImpl.java

```diff
@@ -0,0 +1,78 @@
+package org.sunbird.user.service;
+
+import org.apache.commons.collections.CollectionUtils;
+import org.apache.commons.lang3.StringUtils;
+import org.sunbird.DaoImplType;
+import org.sunbird.exception.BaseException;
+import org.sunbird.request.Request;
+import org.sunbird.response.Response;
+import org.sunbird.user.dao.IUserAddressDao;
+import org.sunbird.user.dao.UserDaoFactory;
+import org.sunbird.util.ProjectLogger;
+import org.sunbird.util.jsonkey.JsonKey;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+/**
+ * This class contains method for processing user address.
+ */
+public class AddressServiceImpl implements IAddressService {
+    @Override
+    public Response createAddress(Request request) throws BaseException {
+        return create(request);
+    }
+
+    private Response create(Request request) {
```

Comment:
Can we take the requestMap only concerning address and not the entire Request object? So that we can do this saveEntity for creating User.